### PR TITLE
Add nextflow io

### PR DESCRIPTION
**Pull request by bj8th**

### CODE REVIEW

**Review by bj8th — COMMENTED** [review]
reviewed all changes again - OK

File: modules/LR_ORFCalling/README.md [hunk 1]
```diff
@@ -76,3 +76,13 @@ For each PB transcript and the set of candidate ORFs as determined by CPAT, sele
 
 ### Threads
 - None
+
+## Script 
+    python orf_calling.py \
+    --orf_coord ../input/jurkat_cpat.ORF_prob.tsv \
+    --gencode ../gencode.v35.annotation.gtf \
+    --sample_gtf ../jurkat_corrected.gtf \
+    --pb_gene ../pb_to_gene.tsv \
```
Comment (gsheynkman) [inline modules/LR_ORFCalling/README.md:85]: Note - pb_gene table from the referencetable module

File: modules/LR_ORFCalling/README.md [hunk 2]
```diff
@@ -76,3 +76,13 @@ For each PB transcript and the set of candidate ORFs as determined by CPAT, sele
...
+    --classification ../jurkat_classification.txt \
+    --sample_fasta ../jurkat_corrected.fasta \
```
Comment (gsheynkman) [inline modules/LR_ORFCalling/README.md:87]: These from sqanti module

File: modules/LR_ORFCalling/orf_calling.sh [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
--g /mnt/shared/ubuntu/session_data/data/input/gencode.v35.annotation.gtf \
--sg /mnt/shared/ubuntu/session_data/data/input/jurkat_corrected.gtf \
--pg /mnt/shared/ubuntu/session_data/data/input/pb_to_gene.tsv \
--c /mnt/shared/ubuntu/session_data/data/input/jurkat_classification.txt \
--sf /mnt/shared/ubuntu/session_data/data/input/jurkat_corrected.fasta \
--o /mnt/shared/ubuntu/session_data/data/output/jurcat_cpat.ORF_called.tsv
+--orf_coord /mnt/shared/ubuntu/session_data/data/input/jurkat_cpat.ORF_prob.tsv \
+--gencode /mnt/shared/ubuntu/session_data/data/input/gencode.v35.annotation.gtf \
+--sample_gtf /mnt/shared/ubuntu/session_data/data/input/jurkat_corrected.gtf \
+--pb_gene /mnt/shared/ubuntu/session_data/data/input/pb_to_gene.tsv \
```
Comment (gsheynkman) [inline modules/LR_ORFCalling/orf_calling.sh:5]: `pb_gene` table now coming from the referencetable module channel

File: modules/LR_ORFCalling/orf_calling.sh [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
+--classification /mnt/shared/ubuntu/session_data/data/input/jurkat_classification.txt \
+--sample_fasta /mnt/shared/ubuntu/session_data/data/input/jurkat_corrected.fasta \
+--output /mnt/shared/ubuntu/session_data/data/output/jurcat_cpat.ORF_called.tsv
```
Comment (gsheynkman) [inline modules/LR_ORFCalling/orf_calling.sh:8]: Suggestion
"""
--output /mnt/shared/ubuntu/session_data/data/output/jurkat_cpat.ORF_called.tsv
"""

File: modules/LR_TranscriptomeSummary/README.md [hunk 5]
```diff
@@ -2,14 +2,16 @@
...
-- fa_file      # gencode fafsa file
-- tpm_file     #
-- yangpolyA    # file prepared manually
-- ribodep_tpm           # expects normalized data
-- pbacc_to_gene_file    # mapping file from Gloria
-- mm_out                # Metamorpheus outputs 
+| argument | description |
+|----------|-------------|
+| gtf_file     | gencode gtf file 
+| squanti_out  | classification.txt file from SQUATI Output
```
Comment (gsheynkman) [inline modules/LR_TranscriptomeSummary/README.md:8]: typo - SQANTI

File: modules/LR_TranscriptomeSummary/README.md [hunk 6]
```diff
@@ -2,14 +2,16 @@
...
-- tpm_file     #
-- yangpolyA    # file prepared manually
-- ribodep_tpm           # expects normalized data
-- pbacc_to_gene_file    # mapping file from Gloria
-- mm_out                # Metamorpheus outputs 
+| argument | description |
+|----------|-------------|
+| gtf_file     | gencode gtf file 
+| squanti_out  | classification.txt file from SQUATI Output 
+| fa_file      | gencode fafsa file
```
Comment (gsheynkman) [inline modules/LR_TranscriptomeSummary/README.md:9]: typo - fasta

File: modules/LR_TranscriptomeSummary/README.md [hunk 7]
```diff
@@ -2,14 +2,16 @@
...
+| tpm_file     |
```
Comment (gsheynkman) [inline modules/LR_TranscriptomeSummary/README.md:10]: description - transcripts per million from kallisto isoform quantification

File: modules/LR_TranscriptomeSummary/README.md [hunk 8]
```diff
@@ -2,14 +2,16 @@
...
+| yangpolyA    | file prepared manually
```
Comment (gsheynkman) [inline modules/LR_TranscriptomeSummary/README.md:11]: Delete - we won't use this file anymore.
This was polyA data on hela cells, and we found that it didn't overlap well the Jurkat data.

File: modules/LR_TranscriptomeSummary/README.md [hunk 9]
```diff
@@ -2,14 +2,16 @@
...
+| ribodep_tpm           | expects normalized data
```
Comment (gsheynkman) [inline modules/LR_TranscriptomeSummary/README.md:12]: list of genes and their TPM in a ribodeplete dataset
This is an optional parameter.

File: modules/LR_TranscriptomeSummary/README.md [hunk 10]
```diff
@@ -2,14 +2,16 @@
...
+| pbacc_to_gene_file    | mapping file from Gloria
```
Comment (gsheynkman) [inline modules/LR_TranscriptomeSummary/README.md:13]: Mapping file from referencetable module.

File: modules/LR_TranscriptomeSummary/README.md [hunk 11]
```diff
@@ -35,4 +37,11 @@
 - None
```
Comment (gsheynkman) [inline modules/LR_TranscriptomeSummary/README.md:37]: delete

File: modules/LR_TranscriptomeSummary/README.md [hunk 12]
```diff
@@ -35,4 +37,11 @@
...
 ## Shell
-- None
+    python transcriptome_summary.py \
+    --sq_out ../input/sq_out.tsv \
+    --tpm ../input/sq_out.tsv \
+    --ribo ../input/sq_out.tsv \
+    --ensg_to_gene ../input/sq_out.tsv \
+    --enst_to_isoname ../input/sq_out.tsv \
+    --len_stats ../input/sq_out.tsv \
+    --odir ../output
```
Comment (gsheynkman) [inline modules/LR_TranscriptomeSummary/README.md:47]: Change to file types

File: modules/LR_TranscriptomeSummary/transcriptome_summary.sh [hunk 13]
```diff
@@ -0,0 +1,8 @@
+python transcriptome_summary.py \
+--sq_out ../input/sq_out.tsv \
+--tpm ../input/sq_out.tsv \
+--ribo ../input/sq_out.tsv \
+--ensg_to_gene ../input/sq_out.tsv \
+--enst_to_isoname ../input/sq_out.tsv \
+--len_stats ../input/sq_out.tsv \
+--odir ../output
```
Comment (gsheynkman) [inline modules/LR_TranscriptomeSummary/transcriptome_summary.sh:8]: Fill in correct filenames.

**Review by gsheynkman — APPROVED** [review]
Before merging, can you make the minor edits under comments.

**Pull request merged**